source/contributing/implementation/debugging/index.rst

```diff
@@ -32,9 +32,8 @@ With these instructions successfully completed, you are ready to import the proj
 
 .. note::
     This is an outline of the steps necessary to setup the workspace. See the `SpongeForge 
```

Comment:
If this sentence covers both SF and SV, this should either say "Sponge" or "SpongeCommon", not "SpongeForge", and it should end with the plural, i.e."implementations".